src/style.css

```diff
@@ -0,0 +1,133 @@
+/* #region CSSreset */
+body {
+  margin: 0;
+}
+/* #endregion */
+
+/* #region Card styles */
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-style: normal;
+}
+
+.card__imagine {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 36px;
+}
+
+.card__label {
+  margin: 0 16px;
+}
+
+.card__label p:first-child {
+  margin: 0;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__label p:last-child {
+  margin: 4px 0 0;
+  margin-bottom: 9px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__rate {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 16px;
+}
+
+.card__feedback {
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 16px;
+}
+
+.card__price p:first-child {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+
+}
+
+.card__price p:last-child {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__buy {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  width: 164px;
+  height: 38px;
+  margin: 0 16px;
+  margin-bottom: 16px;
+  border-radius: 5px;
+  border: 1px solid #00acdc;
+  background: #00acdc;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  color: #fff;
+  transition-delay: 200ms;
+  text-decoration: none;
+}
+
+.card__buy:hover {
+  color: #00acdc ;
+  background: #fff;
+}
+/* #endregion */
+
+/* #region stars */
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
changed just background url, so you can delete background-repeat and background-position

Reply:
fixed.